<!DOCTYPE html>
<html lang="en-uk">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 3.1.1">
  <meta name="generator" content="Hugo 0.51" />
  <meta name="author" content="Dave Clark">

  
  
  
  
    
  
  <meta name="description" content="Quantifying the functional and taxonomic diversity of microbial assemblages is essential to understanding almost all aspects of microbial ecology. In recent years, the advent of Next Generation Sequencing (NGS) technology has accelerated this process. It is now common practice to target and amplify phylogenetic and/or functional marker genes, and use NGS approaches to characterise their diversity across multiple samples. However, all NGS approaches contain inherent methodological biases, producing both the high-quality data required by researchers, and in addition, erroneous sequences and noise. Careful bioinformatic analysis of NGS data is therefore required to quality filter and process sequences in order to avoid misleading inferences from artifactual results. A similar consideration must also be given to any downstream statistical analysis, as an incorrect choice of approach can also produce false conclusions. These various analytical steps and considerations can appear daunting to the uninitiated and may be perceived as a hurdle to completing research. In this chapter, we aim to provide the methods and guidance required to overcome this hurdle, and impart the skills required for a novice bioinformatician to produce a basic analysis of their NGS amplicon data. We focus on data produced using two common NGS technologies, the historically more widely used 454-pyrosequencer, and the currently more widely used Illumina platform. We cover methods for quality filtering and denoising data, picking Operational Taxonomic Units (OTUs), assigning taxonomy to sequences and basic statistical analyses required for hypothesis testing. Implementation of these methods is demonstrated for both of two commonly used pipelines (QIIME and mothur), and additional stand-alone packages (including R), providing the reader with maximum flexibility when analysing their data.">

  
  <link rel="alternate" hreflang="en-uk" href="/publication/dumbrell_ferguson_clark/">

  


  

  

  

  
  
  
  <meta name="theme-color" content="hsl(339, 90%, 68%)">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha256-eSi1q2PG6J7g7ib17yAaWMcrr5GrtohYChqibrV7PBE=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css" integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/dracula.min.css" crossorigin="anonymous">
        
      
    

    

    

  

  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Montserrat:400,700|Roboto:400,400italic,700|Roboto+Mono">
  

  <link rel="stylesheet" href="/styles.css">
  

  
  
  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="Microbial Ecologist">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="Microbial Ecologist">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/publication/dumbrell_ferguson_clark/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@microbeEcology">
  <meta property="twitter:creator" content="@microbeEcology">
  
  <meta property="og:site_name" content="Microbial Ecologist">
  <meta property="og:url" content="/publication/dumbrell_ferguson_clark/">
  <meta property="og:title" content="Microbial community analysis by single-amplicon high-throughput next generation sequencing: data analysis–from raw output to ecology | Microbial Ecologist">
  <meta property="og:description" content="Quantifying the functional and taxonomic diversity of microbial assemblages is essential to understanding almost all aspects of microbial ecology. In recent years, the advent of Next Generation Sequencing (NGS) technology has accelerated this process. It is now common practice to target and amplify phylogenetic and/or functional marker genes, and use NGS approaches to characterise their diversity across multiple samples. However, all NGS approaches contain inherent methodological biases, producing both the high-quality data required by researchers, and in addition, erroneous sequences and noise. Careful bioinformatic analysis of NGS data is therefore required to quality filter and process sequences in order to avoid misleading inferences from artifactual results. A similar consideration must also be given to any downstream statistical analysis, as an incorrect choice of approach can also produce false conclusions. These various analytical steps and considerations can appear daunting to the uninitiated and may be perceived as a hurdle to completing research. In this chapter, we aim to provide the methods and guidance required to overcome this hurdle, and impart the skills required for a novice bioinformatician to produce a basic analysis of their NGS amplicon data. We focus on data produced using two common NGS technologies, the historically more widely used 454-pyrosequencer, and the currently more widely used Illumina platform. We cover methods for quality filtering and denoising data, picking Operational Taxonomic Units (OTUs), assigning taxonomy to sequences and basic statistical analyses required for hypothesis testing. Implementation of these methods is demonstrated for both of two commonly used pipelines (QIIME and mothur), and additional stand-alone packages (including R), providing the reader with maximum flexibility when analysing their data.">
  
  
    
  <meta property="og:image" content="/img/icon-192.png">
  <meta property="og:locale" content="en-uk">
  
  <meta property="article:published_time" content="2016-11-01T00:00:00&#43;00:00">
  
  <meta property="article:modified_time" content="2019-02-11T07:24:53&#43;00:00">
  

  

  

  <title>Microbial community analysis by single-amplicon high-throughput next generation sequencing: data analysis–from raw output to ecology | Microbial Ecologist</title>

</head>
<body id="top" data-spy="scroll" data-target="#TableOfContents" data-offset="71" class="dark">
  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header fixed-top">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" role="textbox" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


<nav class="navbar navbar-light fixed-top navbar-expand-lg py-0" id="navbar-main">
  <div class="container">

    
      <a class="navbar-brand" href="/">Microbial Ecologist</a>
      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
        <span><i class="fas fa-bars"></i></span>
      </button>
      

    
    <div class="collapse navbar-collapse" id="navbar">

      
      
      <ul class="navbar-nav ml-auto">
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#about">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#publications">
            
            <span>Publications</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#posts">
            
            <span>Posts</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/fossils/">
            
            <span>Fossils</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        

      

        

        
        <li class="nav-item">
          <a class="nav-link js-search" href="#"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        
      </ul>

    </div>
  </div>
</nav>

<div class="pub" itemscope itemtype="http://schema.org/CreativeWork">

  












  

  
  
  
<div class="article-container pt-3">
  <h1 itemprop="name">Microbial community analysis by single-amplicon high-throughput next generation sequencing: data analysis–from raw output to ecology</h1>

  

  
    

<div class="article-metadata">

  
  
  
  <div>
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Alex J Dumbrell</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Robert MW Ferguson</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name"><strong>Dave R Clark</strong></span>
    </span>
    
  </div>
  

  <span class="article-date">
    
        Last updated on
    
    <meta content="2016-11-01 00:00:00 &#43;0000 GMT" itemprop="datePublished">
    <time datetime="2019-02-11 07:24:53 &#43;0000 GMT" itemprop="dateModified">
      Feb 11, 2019
    </time>
  </span>
  <span itemscope itemprop="publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Dave Clark">
  </span>

  

  
  

  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=Microbial%20community%20analysis%20by%20single-amplicon%20high-throughput%20next%20generation%20sequencing%3a%20data%20analysis%e2%80%93from%20raw%20output%20to%20ecology&amp;url=%2fpublication%2fdumbrell_ferguson_clark%2f"
         target="_blank" rel="noopener">
        <i class="fab fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=%2fpublication%2fdumbrell_ferguson_clark%2f"
         target="_blank" rel="noopener">
        <i class="fab fa-facebook-f"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=%2fpublication%2fdumbrell_ferguson_clark%2f&amp;title=Microbial%20community%20analysis%20by%20single-amplicon%20high-throughput%20next%20generation%20sequencing%3a%20data%20analysis%e2%80%93from%20raw%20output%20to%20ecology"
         target="_blank" rel="noopener">
        <i class="fab fa-linkedin-in"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=%2fpublication%2fdumbrell_ferguson_clark%2f&amp;title=Microbial%20community%20analysis%20by%20single-amplicon%20high-throughput%20next%20generation%20sequencing%3a%20data%20analysis%e2%80%93from%20raw%20output%20to%20ecology"
         target="_blank" rel="noopener">
        <i class="fab fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=Microbial%20community%20analysis%20by%20single-amplicon%20high-throughput%20next%20generation%20sequencing%3a%20data%20analysis%e2%80%93from%20raw%20output%20to%20ecology&amp;body=%2fpublication%2fdumbrell_ferguson_clark%2f">
        <i class="fas fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>

    







  








<div class="btn-links mb-3">
  
  






  



  
    
  


<a class="btn btn-outline-primary my-1 mr-1" href="https://link.springer.com/protocol/10.1007/8623_2016_228" target="_blank" rel="noopener">
  PDF
</a>

















</div>


  
</div>



  <div class="article-container">

    
    <h3>Abstract</h3>
    <p class="pub-abstract" itemprop="text">Quantifying the functional and taxonomic diversity of microbial assemblages is essential to understanding almost all aspects of microbial ecology. In recent years, the advent of Next Generation Sequencing (NGS) technology has accelerated this process. It is now common practice to target and amplify phylogenetic and/or functional marker genes, and use NGS approaches to characterise their diversity across multiple samples. However, all NGS approaches contain inherent methodological biases, producing both the high-quality data required by researchers, and in addition, erroneous sequences and noise. Careful bioinformatic analysis of NGS data is therefore required to quality filter and process sequences in order to avoid misleading inferences from artifactual results. A similar consideration must also be given to any downstream statistical analysis, as an incorrect choice of approach can also produce false conclusions. These various analytical steps and considerations can appear daunting to the uninitiated and may be perceived as a hurdle to completing research. In this chapter, we aim to provide the methods and guidance required to overcome this hurdle, and impart the skills required for a novice bioinformatician to produce a basic analysis of their NGS amplicon data. We focus on data produced using two common NGS technologies, the historically more widely used 454-pyrosequencer, and the currently more widely used Illumina platform. We cover methods for quality filtering and denoising data, picking Operational Taxonomic Units (OTUs), assigning taxonomy to sequences and basic statistical analyses required for hypothesis testing. Implementation of these methods is demonstrated for both of two commonly used pipelines (QIIME and mothur), and additional stand-alone packages (including R), providing the reader with maximum flexibility when analysing their data.</p>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Type</div>
          <div class="col-12 col-md-9">
            
            <a href="/publication/#6">
              Book section
            </a>
            
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Publication</div>
          <div class="col-12 col-md-9">In <em>Hydrocarbon and Lipid Microbiology Protocols</em>.</div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Date</div>
          <div class="col-12 col-md-9" itemprop="datePublished">
            November, 2016
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-sm-none space-below"></div>

    <div class="space-below"></div>

    <div class="article-style"></div>

    


<div class="article-tags">
  
  <a class="badge badge-light" href="/tags/16s-rrna/">16S rRNA</a>
  
  <a class="badge badge-light" href="/tags/454-pyrosequencing/">454-Pyrosequencing</a>
  
  <a class="badge badge-light" href="/tags/amplicon/">Amplicon</a>
  
  <a class="badge badge-light" href="/tags/bioinformatics/">Bioinformatics</a>
  
  <a class="badge badge-light" href="/tags/biolinux/">BioLinux</a>
  
  <a class="badge badge-light" href="/tags/illumina/">Illumina</a>
  
  <a class="badge badge-light" href="/tags/microbial-ecology/">Microbial ecology</a>
  
  <a class="badge badge-light" href="/tags/mothur/">Mothur</a>
  
  <a class="badge badge-light" href="/tags/next-generation-sequencing/">Next generation sequencing</a>
  
  <a class="badge badge-light" href="/tags/qiime/">QIIME</a>
  
</div>



    



  







  </div>
</div>



<div class="container">
  <footer class="site-footer">
  
  <p class="powered-by">
    <a href="/privacy/">Privacy Policy</a>
  </p>
  

  <p class="powered-by">
    &copy; 2018 &middot; 

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" id="back_to_top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

</div>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha512-+NqPlbbtM1QqiK8ZAo4Yrj2c4lNQoGv8P79DPtKzj++l5jnN39rHA/xsqn8zE9l0uSoxaCdrOgFs6yjyfbBxSg==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha256-VsEqElsCHSGmnmHXGQzvoWjWwoznFSZc6hs7ARLRacQ=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
        
      

      
      
    

    
    

    
    
    

    
    
    <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script>
      const search_index_filename = "/index.json";
      const i18n = {
        'placeholder': "Search...",
        'results': "results found",
        'no_results': "No results found"
      };
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    
    
    
    
    
    
    
    <script src="/js/academic.min.74b0d3361475b08c40cf31120e82937b.js"></script>

    

  </body>
</html>

